include/aca_util.h

```diff
@@ -116,7 +116,7 @@ static inline bool aca_validate_mac_address(const char *mac_string)
   unsigned char mac[6];
 
   if (mac_string == nullptr) {
-    ACA_LOG_ERROR("Input mac_string is null\n");
+    ACA_LOG_ERROR("%s", "Input mac_string is null\n");
```

Comment:
what is the compiler warning for this? the change looks a bit weird...

Note that we plan to update the logging system in ACA tracked by #116